app/controllers/users_controller.rb

```diff
@@ -4,25 +4,29 @@ class UsersController < ApplicationController
   before_action :get_user_by_id, only: [:show, :edit, :update]
 
   def index
-    @pagy, @users = pagy(User.all)
+    @pagy, @users = pagy(User.where(activated: true))
   end
 
   def new
     @user = User.new
   end
 
   def show
+    redirect_to root_url and return unless @user.activated?
```

Comment:
dùng `&&` hoặc dạng này em nhé
```suggestion
    return redirect_to root_url unless @user.activated?
```

Reply:
Em nghĩ nếu bỏ keyword `return` như dưới đây thì logic cũng giống nhỉ anh?
```rb
redirect_to root_url unless @user.activated?
```